spec/models/enterprise_fee_spec.rb

```diff
@@ -179,9 +165,26 @@
                                  label: 'hello' })
 
       expect do
-        EnterpriseFee.clear_all_adjustments order
+        described_class.clear_order_adjustments order
       end.to change { order.adjustments.count }.by(0)
     end
+
+    it "doesn't clear adjustments from many fees and on all line items" do
+      line_item1 = create(:line_item, order:, variant: order_cycle.variants.first)
+      line_item2 = create(:line_item, order:, variant: order_cycle.variants.second)
+
+      # Order adjustment
+      order_cycle.coordinator_fees[0].create_adjustment('foo1', line_item1.order, true)
+      order_cycle.coordinator_fees[0].create_adjustment('foo2', line_item2.order, true)
+      # Line item adjustment
+      order_cycle.exchanges[0].enterprise_fees[0].create_adjustment('foo3', line_item1, true)
+      order_cycle.exchanges[0].enterprise_fees[0].create_adjustment('foo4', line_item2, true)
+
+      # does not clear line item adjustments
+      expect do
+        described_class.clear_order_adjustments order
+      end.to change { order.all_adjustments.count }.by(-2)
```

Comment:
This doesn't check _which adjustments_ were cleared. Can we `expect(order.all_adjustments).to_not include foo1, foo2`?